### DISCUSSION TITLE
[Q] Need help in custom data validation

### DISCUSSION THREAD

**Discussion by sumitbhavra** [exceljs/exceljs#2433] [Q&A]
## 💬 
Hi, I need help with custom data validation. I need to stop users entering any special charter including % in a cell. I need only a numeric value. could you please provide custom validation examples. I need the cell value to be in a range. I get a min and max value. I want the cell value in between.

Please help me out. This bug came in production and business is running behind me.

Regards
Sumit Singh

**Comment by mattdevo** [comment]
@sumitbhavra have you found a solution for this? I am also having issues using the custom data validation
